.github/.wordlist.txt

```diff
@@ -1157,6 +1157,7 @@ rmw
 rodata
 Rollershade
 rootfs
+rootshell
```

Comment:
switch quoting to backtics so this is not needed as a word

Reply:
Done